client/lib/wpcom-undocumented/lib/undocumented.js

```diff
@@ -1601,6 +1601,30 @@ Undocumented.prototype.installThemeOnJetpack = function( siteId, themeId, fn ) {
 	);
 };
 
+/**
+ * Updates a theme from WordPress.org or WordPress.com on the given Jetpack site.
+ *
+ * @param {String}    siteId    The site ID
+ * @param {String}    themeId Theme slug, suffixed with -wpcom if it's a WordPress.com theme.
+ * @param {Function}  fn        The callback function
+ * @returns {Promise} promise
+ */
+Undocumented.prototype.updateThemeOnJetpack = function( siteId, themeId, fn ) {
+	const path = `/sites/${ siteId }/themes`;
+	debug( path );
+
+	return this.wpcom.req.post(
+		{
+			path,
+			body: {
+				action: 'update',
+				themes: themeId,
+			},
+		},
+		fn
+	);
+};
```

Comment:
Unfortunately we have intentionally abandoned adding code to `wpcom().undocumented`

Did you see documentation that still indicates that we should?

Reply:
Thanks for the review. I just followed what was there for `installTheme`.